lib/apress/selenium_products/spec/company_site/product_creation_page/new_form_spec.rb

```diff
@@ -0,0 +1,633 @@
+    @product_page          = ProductPage.new
+    @product_creation_page = CompanySite::ProductCreationPage.new
+
+    log_in_as(:admin)
+  end
+
+  after(:all) { log_out }
+
+  context 'Когда переходим по урлу .../products/new' do
+    before do
+      navigate_to_product_creation_page
+      wait_until { @product_creation_page.page_title? }
+    end
+
+    it 'выводится заголовок Создание товара или услуги' do
+      expect(@product_creation_page.page_title?).to be_truthy
+    end
+  end
+
+  describe 'Когда заполняем поля и создаем товар' do
+    before(:all) { navigate_to_product_creation_page }
+
+    context 'когда ни одно поле не заполнено' do
+      before { @product_creation_page.save }
+
+      it 'отобразится ошибка сохранения под полем Название' do
+        expect(@product_creation_page.name_error?).to be_truthy
+      end
+    end
+
+    context 'когда все поля заполнены максимальными значениями' do
+      before(:all) do
+        @options = {
+          name: Faker::Lorem.paragraph_by_chars(number: 120),
+          article: Faker::Lorem.paragraph_by_chars(number: 128),
+          exists: CONFIG['product_creation']['exists']['in_stock'],
+          price: Faker::Number.number(digits: 10),
+          currency: CONFIG['product_creation']['currency']['usd'],
+          measure: CONFIG['product_creation']['measure']['box'],
+          wholesale_price: Faker::Number.number(digits: 10),
+          wholesale_qty: Faker::Number.number(digits: 6),
+          short_description: Faker::Lorem.paragraph_by_chars(number: 255),
+          description: CONFIG['product_creation']['full_description'],
+          tag_title: Faker::Lorem.characters(number: 255),
+          rubric: CONFIG['rubrics_names_by_level'][5],
+          priority_lvl4: true,
+          priority_lvl5: true,
+          company_traits: {
+            CONFIG['product_creation']['company_traits']['traits'][1] => CONFIG['product_creation']['company_traits']['values'][1],
+            CONFIG['product_creation']['company_traits']['traits'][2] => CONFIG['product_creation']['company_traits']['values'][2],
+          },
+          portal_traits: {
+            CONFIG['product_creation']['portal_traits']['traits'][1] => CONFIG['product_creation']['portal_traits']['values'][1],
+            CONFIG['product_creation']['portal_traits']['traits'][2] => CONFIG['product_creation']['portal_traits']['values'][2],
+          },
+          path_to_image: IMAGE_PATH,
+          qty_of_images: 4, # на самом деле будет загружено 10 изображений
+          description_of_photo: Faker::Lorem.paragraph_by_chars(number: 255),
+          group: CONFIG['product_creation']['group'],
+          path_to_doc: IMAGE_PATH,
+          qty_of_doc: 3, # на самом деле будет загружено 5 документов - это максимум
+          soputka: CONFIG['product_creation']['soputka'],
+          qty_of_soputka: 4
+        }
+
+        # На БЛ нет этого поля, только на ПЦ
+        if @product_creation_page.min_qty_input?
```

Comment:
[Correctable] Style/IfUnlessModifier: Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.